### DISCUSSION TITLE
I want to know when Java will be supported

### DISCUSSION THREAD

**Discussion by ArvinLiu118** [icerpc/icerpc-csharp#4091] [General]
### Desired feature

Java is widely used, and I hope it will be supported soon. Thank you.

### Problem

Java is widely used, and I hope it will be supported soon. Thank you.

### Alternatives

Java is widely used, and I hope it will be supported soon. Thank you.

### Additional context

Java is widely used, and I hope it will be supported soon. Thank you.

**Comment by pepone** [comment]
Hi,

We don't have an immediate plan for a Java port, we are currently working on getting a Rust port.
